src/common/formUtils.ts

```diff
@@ -0,0 +1,46 @@
+import * as Yup from "yup";
+
+/**
+ * Password regex, it must contain the following:
+ * 8 characters, 1 uppercase letter, 1 lowercase letter and 1 number.
+ */
+const passwordRequirementRegex = /^(?=.*[0-9])(?=.*[a-z])(?=.*[A-Z]).{8,}$/;
+
+export const commonYupValidation = {
```

Comment:
Since we are reusing these validations in multiple places and potentially more... i created this and we can expand on it, for example:

```
import { commonYupValidation } from "common";
...
  const validationSchema = Yup.object({
    firstName: commonYupValidation.firstName.matches(/^[aA-zZ\s]+$/, "Please only use letters"),
  });
...
```